### DISCUSSION TITLE
2021-32: 团队每日例会的变迁

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#236] [Announcements]
# 2021-32: 团队每日例会的变迁

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2021-32/

**Comment by howbbig** [comment]
请问团队多少人？每天都开会吗？

**Reply by Xuanwo** [reply]
团队现在五个人，每天都开~

**Comment by youzeliang** [comment]
感觉每天开还是没啥必要

**Reply by Xuanwo** [reply]
我们是纯远程团队，没有例会的话我们每天基本上没啥沟通了 = =

**Reply by youzeliang** [reply]
额、那就理解了

**Comment by grokking88** [comment]
请问不讨论细节的话是如何resolve那些blocker的？

**Reply by Xuanwo** [reply]
block 我们会当场创建 issue 来 track，如果只是一些理解的问题我们第二天会安排一个视频会议
